jekyll/_cci2/jobs-steps.md

```diff
@@ -1,17 +1,21 @@
 ---
 layout: classic-docs
-title: "Jobs, Steps, and Workflows"
-short-title: "Jobs, Steps, and Workflows"
+title: "Orbs, Jobs, Steps, and Workflows"
+short-title: "Orbs, Jobs, Steps, and Workflows"
 description: "Description of Jobs and Steps"
 categories: [migration]
 order: 2
 ---
 
-The document provides an overview of Jobs and Steps, new [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) keys that are nested inside Steps, and new keys for Workflows.
+The document provides an overview of Jobs, Steps, Workflows and new [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) keys for Orbs.
```

Comment:
Is it standard to use terms like "new keys"? How long do we consider it explicitly new at a docs level?

Reply:
yep